.pipelines/cni/singletenancy/windows-cni-load-test-template.yaml

```diff
@@ -198,17 +106,23 @@ stages:
               scriptType: "bash"
               addSpnToEnvironment: true
               inlineScript: |
-                set -ex
-                if [ "$(DELETE_RESOURCES)" ]
-                then
-                  echo "Deleting Cluster and resource group"
-                  make -C ./hack/aks set-kubeconf AZCLI=az CLUSTER=${{ parameters.clusterName }}-$(make revision)
-                  make -C ./hack/aks azcfg AZCLI=az REGION=$(LOCATION)
-                  make -C ./hack/aks down AZCLI=az REGION=$(LOCATION) SUB=$(SUB_AZURE_NETWORK_AGENT_BUILD_VALIDATIONS) CLUSTER=${{ parameters.clusterName }}-$(make revision)
-                  echo "Cluster and resources down"
-                else
-                  echo "Deletion of resources is False"
-                fi
-            name: "CleanUpCluster"
-            displayName: "Cleanup cluster"
-    condition: always()
+                echo "Delete load-test Namespace"
+                make -C ./hack/aks set-kubeconf AZCLI=az CLUSTER=${{ parameters.clusterName }}-$(commitID)
+                kubectl delete ns load-test

```

Comment:
wondering is this causing problem for e2e  that is why we deleted the load test namespace.?

Reply:
It is being deleted as after this CNI k8s E2E requires space on each node to run its tests. No tests are being run after this that require the use or leverage having pods active on each node.